applications/serial_lte_modem/boards/nrf9131ek_nrf9131_ns.conf

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Configuration file for nRF9131EK.
+# This file is merged with prj.conf in the application folder, and options
+# set here will take precedence if they are present in both files.
+
+# Use UART_0 (when working with PC terminal)
```

Comment:
These comments should be removed as they apply to a scenario where UART2 is used. Which is, as far as I understand, not a scenario with this board.